### ISSUE TITLE
Unable to configure with steps in readme 

### ISSUE THREAD

**Issue by alexkutsan** [malirod/cppecho#9]
```
➜  cppecho git:(master) echo $BOOST_HOME
/home/alex/pro/boost-bin/build
➜  cppecho git:(master) ls /home/alex/pro/boost-bin/build 
include  lib
➜  cppecho git:(master) ./waf configure build            
Setting top to                           : /home/alex/pro/cppecho 
Setting out to                           : /home/alex/pro/cppecho/build 
Checking for 'clang++' (C++ compiler)    : /usr/bin/clang++ 
sed -i "s/touch configure/touch configure && sync/" ./scripts/fix-timestamps.sh
sed: can't read ./scripts/fix-timestamps.sh: No such file or directory
Command failed with exit code 2
(complete log in /home/alex/pro/cppecho/build/config.log)

```

**Comment by alexkutsan** [comment]
Just add to README that this repo contains submodules and it should be inited and updated.

**Comment by malirod** [comment]
@alexkutsan, did you run `waf configure`? this command should be executed first and it will do `git submodule --init --recursive` if logger is not found

**Comment by malirod** [comment]
Resolved by #10
